packages/react-ui-components/src/Button/style.css

```diff
@@ -36,10 +36,12 @@
     padding: 0 8px;
 }
 
-.btn--clean,
-.btn--clean i {
+.btn--clean {
     background-color: var(--brandColorsContrastDarker);
 }
+.btn--clean i {
+    background-color: 0;
```

Comment:
sorry for knit-picking, but why is it even needed to set a transparent background? It should be transparent by default, no?!